### PR TITLE
Update version and changelog

### PR DESCRIPTION
**Pull request by olafura**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # KitchenSink 1.3.0 Release Notes (2017-08-25)
 
+## Changes since 1.3.0
```
Comment (mikestok) [inline CHANGELOG.md]: Should line 1 be

    # KitchenSink 1.3.1 Release Notes (2017-08-28)

... or even deleted?

Comment (olafura) [reply CHANGELOG.md]: I would say deleted

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+  * Added KitchenSink.Integer
+    * ensure_integer/1
```
Comment (mariusbutuc) [inline CHANGELOG.md]: Worth mentioning https://github.com/planswell/kitchen-sink/commit/455c8d5a5583787db5afc9d4ccf3a0b2bd9832aa as well?

Comment (olafura) [reply CHANGELOG.md]: Done ;)

**Review by mariusbutuc — APPROVED** [review]

**Review by Corvillus — APPROVED** [review]

**Pull request merged**